### PR TITLE
Bug/gsye 55

### PR DESCRIPTION
**Pull request by spyrostz**

### CODE REVIEW

**Review by hannesdiedrich — APPROVED** [review]
Spent some time to understand. Looks clear now. Great re-use of ExternalTickCounter. Thanks a lot for dealing with this!

File: src/gsy_e/models/myco_matcher/myco_external_matcher.py [hunk 1]
```diff
@@ -42,71 +46,84 @@ class ExternalMatcherEventsEnum(Enum):
...
+        self._channel_prefix = f"external-myco/{self.simulation_id}"
+        self._events_channel = f"{self._channel_prefix}/events/"
+        self._setup_redis_connection()
 
     def _setup_redis_connection(self):
         self.myco_ext_conn = ResettableCommunicator()
         self.myco_ext_conn.sub_to_multiple_channels(
             {"external-myco/simulation-id/": self.publish_simulation_id,
-             f"{self._channel_prefix}/offers-bids/": self.publish_orders,
+             f"{self._channel_prefix}/offers-bids/": self._publish_orders_message_buffer.append,
```
Comment (Hassan754) [inline src/gsy_e/models/myco_matcher/myco_external_matcher.py:75]: Nice, I didn't know we could do this

**Review by Hassan754 — APPROVED** [review]
LGTM

File: src/gsy_e/models/myco_matcher/myco_external_matcher.py (hunk 1 above)
Comment (spyrostz) [reply src/gsy_e/models/myco_matcher/myco_external_matcher.py:75]: Me neither, pylint was helpful here

**Pull request merged**